configuration/locationtagmaps/locationtagmaps-site-lacolline.csv

```diff
@@ -1,2 +1,23 @@
+La Colline,,,,,,,,,,,,,,TRUE,,,,,,TRUE,,,,,,,TRUE,,,,,,,,TRUE,
+Achiv Santral | La Colline,,,,,,,,,,,,,,,TRUE,,,TRUE,,,,,,,,,,,TRUE,,,,,,,
+COVID-19 | Izolman | La Colline,TRUE,,,,TRUE,TRUE,,,,,,,,,,TRUE,TRUE,TRUE,,,,,,,TRUE,,,,TRUE,,,TRUE,TRUE,,,TRUE
+Ijans | La Colline,,,,,TRUE,TRUE,,,,TRUE,TRUE,TRUE,,,TRUE,TRUE,,TRUE,,,,,,TRUE,TRUE,,,,,TRUE,,,TRUE,,,TRUE
+Klinik Ekstèn | La Colline,,,TRUE,,TRUE,TRUE,,TRUE,,TRUE,,,TRUE,,TRUE,TRUE,TRUE,TRUE,,,TRUE,TRUE,,TRUE,TRUE,,,TRUE,,TRUE,,,TRUE,,,TRUE
+Klinik Ekstèn Jeneral | La Colline,,,TRUE,,,TRUE,,,,,,,,,TRUE,TRUE,TRUE,TRUE,,,,,,,,TRUE,,TRUE,TRUE,TRUE,,,,,,TRUE
+Klinik Ekstèn Famasi | La Colline,,,TRUE,,,,,,TRUE,,,,,,,,,TRUE,,,,,,,,,,,,,,,,,,
+Onkoloji | La Colline,TRUE,,TRUE,,,TRUE,TRUE,TRUE,,,,,,,,TRUE,TRUE,TRUE,,,,,TRUE,TRUE,TRUE,,,,,,,,,,,TRUE
+Sal Aprè Akouchman | La Colline,TRUE,TRUE,,,TRUE,TRUE,,TRUE,,,,,,,TRUE,TRUE,,TRUE,TRUE,,TRUE,,,TRUE,TRUE,,,TRUE,TRUE,,TRUE,,TRUE,,,TRUE
+Sal Aprè Operasyon | La Colline,TRUE,TRUE,,,TRUE,TRUE,,TRUE,,,,,,,TRUE,TRUE,,TRUE,,,,,,TRUE,TRUE,,,,,TRUE,TRUE,,TRUE,,,TRUE
+Sal Aprè Operasyon | Sante Fanm | La Colline,TRUE,TRUE,,,TRUE,,,TRUE,,,,,,,TRUE,,,TRUE,,,,,,TRUE,TRUE,,,,,TRUE,TRUE,,TRUE,,,TRUE
+Sal Avan Akouchman | La Colline,TRUE,TRUE,,,TRUE,TRUE,,TRUE,,,,,,,TRUE,TRUE,,TRUE,TRUE,,TRUE,,,TRUE,TRUE,,,TRUE,TRUE,,TRUE,,TRUE,,,TRUE
+Sal Fanm | La Colline,TRUE,TRUE,,,TRUE,,,TRUE,,TRUE,,,,,TRUE,TRUE,,TRUE,,,,,,TRUE,TRUE,,,,,,,,TRUE,,,TRUE
+Sal Gason | La Colline,TRUE,TRUE,,,TRUE,,,TRUE,,TRUE,,,,,TRUE,TRUE,,TRUE,,,,,,TRUE,TRUE,,,,,,,,TRUE,,,TRUE
+Sal Izolman | La Colline,TRUE,TRUE,,,TRUE,,,TRUE,,TRUE,,,,,TRUE,TRUE,,TRUE,,,,,,TRUE,TRUE,,,,,,,,TRUE,,,
+Sal Reyabilitasyon | La Colline,TRUE,TRUE,,,TRUE,,,TRUE,,,,,,,TRUE,TRUE,,TRUE,,,,,,TRUE,TRUE,,,,,,,,TRUE,,,TRUE
+Sal Timoun | La Colline,TRUE,TRUE,,,TRUE,,,TRUE,,,,,,,TRUE,TRUE,TRUE,TRUE,TRUE,,TRUE,,,TRUE,TRUE,TRUE,,TRUE,,,,,TRUE,,,TRUE
+Sante Fanm | La Colline,,,TRUE,,TRUE,TRUE,,TRUE,,,,,TRUE,,TRUE,TRUE,TRUE,TRUE,TRUE,,TRUE,,,TRUE,TRUE,,,TRUE,,,,,TRUE,TRUE,,TRUE
+Swen Entansif | La Colline,TRUE,TRUE,,,TRUE,,,TRUE,,TRUE,,,,,TRUE,TRUE,,TRUE,,,,,,TRUE,TRUE,,,,,,,,TRUE,,,TRUE
+Swen Entansif Neonatal | La Colline,TRUE,TRUE,,,TRUE,,,TRUE,,,,,,,TRUE,TRUE,,TRUE,TRUE,,,,,TRUE,TRUE,,,TRUE,,,,,TRUE,,,TRUE
+Travay e Akouchman | La Colline,TRUE,TRUE,,,TRUE,,,TRUE,,,,,,,TRUE,TRUE,,TRUE,TRUE,,TRUE,,,TRUE,TRUE,,,TRUE,,,,,TRUE,,,TRUE
+UMI | La Colline,TRUE,TRUE,,,TRUE,TRUE,,TRUE,,,,,,,,TRUE,TRUE,TRUE,,,,,,,TRUE,,,,TRUE,,,TRUE,TRUE,,,TRUE
```

Comment:
I just copied the locationtagmap file for Thomonde and then renamed "Thomonde" -> "La Colline"